response_operations_ui/controllers/message_controllers.py

```diff
@@ -36,12 +36,10 @@ def get_conversation(thread_id):
         raise ApiError(response)
 
 
-def get_conversation_count(params):
+def get_conversation_count(survey_id, conversation_tab, business_id):
     logger.info("Retrieving count of threads")
```

Comment:
I know you didn't add this line, but can you add some metadata to the log line like you did in the other function)